src/common/browser-adapters/chrome-adapter.ts

```diff
@@ -177,4 +178,20 @@ export class ChromeAdapter implements BrowserAdapter, StorageAdapter, CommandsAd
     public getUrl(urlPart: string): string {
         return chrome.extension.getURL(urlPart);
     }
+
+    public requestPermissions(permissions: Permissions.Permissions): Promise<boolean> {
+        return browser.permissions.request(permissions);
+    }
+
+    public addListenerOnPermissionsAdded(callback: (permissions: Permissions.Permissions) => void): void {
+        chrome.permissions.onAdded.addListener(callback);
```

Comment:
Why `chrome` and not `browser` for the `addListener` calls? They should have the same signature